### ISSUE TITLE
Exception when running the cli against a local repository

### ISSUE THREAD

**Issue by javaguruen** [moderneinc/springboot-migration-workshop#23]
Running the following command against my local repositury fails with exception:

command:
> mod build .

version: Moderne CLI v0.3.2

Output:
Where:
java.lang.NullPointerException: Cannot invoke "java.util.Map.values()" because the return value of "io.moderne.commons.graphql.GraphqlResponse.getData()" is null
  io.moderne.cli.GraphqlRequest.response(GraphqlRequest.java:58)
  io.moderne.cli.buildtools.ModerneDownload.findCached(ModerneDownload.java:108)
  io.moderne.cli.commands.Build.build(Build.java:98)
  io.moderne.cli.commands.Build.run(Build.java:76)

* What went wrong:
Cannot invoke "java.util.Map.values()" because the return value of "io.moderne.commons.graphql.GraphqlResponse.getData()" is null

**Comment by javaguruen** [comment]
May be related, but also had problem installing recipes:

```
mod config recipes install --from-moderne
    ▛▀▀▚▖  ▗▄▟▜
    ▌   ▜▄▟▀  ▐
    ▛▀▀█▀▛▀▀▀▀▜
    ▌▟▀  ▛▀▀▀▀▜
    ▀▀▀▀▀▀▀▀▀▀▀
Moderne CLI v0.3.2

> Listing recipe artifacts from Moderne

Moderne has 38 recipe artifacts.

> Resolving recipes from artifacts

> org.openrewrite:rewrite-cobol:2.13.0-SNAPSHOT
Done (0.74s)


FAILURE: mod failed with an exception.

* Where:
io.moderne.commons.FatalException: Failed to resolve recipe dependencies.
  io.moderne.commons.recipe.RecipeLoader.lambda$artifacts$6(RecipeLoader.java:183)
  java.util.Optional.orElseThrow(Optional.java:290)
  io.moderne.commons.recipe.RecipeLoader.artifacts(RecipeLoader.java:183)
  io.moderne.commons.recipe.RecipeLoader.artifacts(RecipeLoader.java:173)
  io.moderne.commons.recipe.RecipeLoader.list(RecipeLoader.java:42)
  io.moderne.cli.commands.config.Recipes$Install.installFromModerne(Recipes.java:107)
  io.moderne.cli.commands.config.Recipes$Install.run(Recipes.java:61)

* What went wrong:
Failed to resolve recipe dependencies.

* Try:
> Report to support@moderne.io
```

**Comment by timtebeek** [comment]
Thanks for reporting these issues! They've since been resolved with a new v1.0.8 of the Moderne CLI.
